tests/layer_tests/tensorflow_tests/test_tf_ArgMinMax.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+
+from common.tf_layer_test_class import CommonTFLayerTest
+
+# Testing operation ArgMin, ArgMax (Initial Implementation)
+# Documentation: https://www.tensorflow.org/api_docs/python/tf/raw_ops/ArgMin
+#                https://www.tensorflow.org/api_docs/python/tf/raw_ops/ArgMax
+
+class TestArgMinMax(CommonTFLayerTest):
+    # input_shape - should be an array
+    # dimension - dimension to be used, for vector should be 0
+    # op_type - type of testing operation
+    # ir_version - common parameter
+    # use_new_frontend - common parameter
+    def create_argminmax_placeholder_const_net(self, input_shape, dimension, op_type, ir_version, use_new_frontend):
+        """
+            Tensorflow net                  IR net
+
+            Placeholder->op_type    =>       Placeholder->TopK->Squeeze
+                         /                               /     /
+            Const-------/                   Const-------/-----/
+
+        """
+
```

Comment:
please remove it. It looks weird

Reply:
I'm not sure I understand your current approach. In all older tests I see mention of graph for input and output models.
When should it be removed in new tests? Should it be removed for older tests? Why should it be removed?
Btw, when I investigated tests first time, it was a great reference for understanding layer tests.